config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -0,0 +1,69 @@
+presets:
+- labels:
+    preset-azure-acsengine: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: CLOUD_CONFIG
+    value: random 
+  - name: AZ_STORAGE_CONTAINER_NAME
+    value: mystoragecontainer
+  - name: REGISTRY
+    value: gcr.io/win-e2e-test/win-e2e-hyperkube
+  - name: WIN_BUILD
+    value: https://raw.githubusercontent.com/adelina-t/acs-engine/change_gen_script_for_e2e/scripts/build-windows-k8s.sh
```

Comment:
personal repository?

Reply:
oups it's an artifact leftover from before merging build script into upstream acs-engine. It's the exact same script, but I changed it to use the acs-engine upstream one as it is merged.